prisma/schema.prisma

```diff
@@ -5,7 +5,7 @@ generator client {
 datasource db {
   provider          = "postgresql"
   url               = env("DATABASE_URL")
-  shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
+  shadowDatabaseUrl = env("HEROKU_POSTGRESQL_CYAN_URL")
 }
 
 model User {
```

Comment:
Hey Jalaj
What's this file for? I don't know that's why asking

Reply:
So I am building the backend myself and I am using prisma, this is the schema file. Think of it as all the tables and columns present in the DB.